src/main/java/io/sentry/telemetry/SentryTelemetryService.java

```diff
@@ -74,7 +74,7 @@ public void start(
                 pluginManager);
         final @Nullable InfoOutput infoOutput = determineSentryCliInfo(cliRunner, pluginConfig);
 
-        if (infoOutput != null && infoOutput.isSaas) {
+        if (infoOutput != null && !infoOutput.isSaas) {
```

Comment:
```suggestion
        if (infoOutput != null && infoOutput.isSaas) {
```

Reply:
was this just for testing?

Reply:
It should actually be a not check, since it disabled telemetry with message:

> running against a self hosted instance.

In SAGP we check `if (startParameters.saas == false) {`

Reply:
oh missed the message part, looks good then!